tests/jerry/es2015/typedarray-prototype-copy-within.js

```diff
@@ -0,0 +1,73 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var normal_typedarras = [
+    new Uint8Array([0, 1, 2, 3, 4, 5]),
+    new Uint16Array([0, 1, 2, 3, 4, 5]),
+    new Uint32Array([0, 1, 2, 3, 4, 5]),
+    new Float32Array([0, 1, 2, 3, 4, 5]),
+    new Float64Array([0, 1, 2, 3, 4, 5]),
+    new Int8Array([0, 1, 2, 3, 4, 5]),
+    new Int16Array([0, 1, 2, 3, 4, 5]),
+    new Int32Array([0, 1, 2, 3, 4, 5])
+  ];
+
+normal_typedarras.forEach(function(e){
+    try {
+        e.prototype.copyWithin.call (undefined);
+        assert (false);
+    } catch (e) {
```

Comment:
If the functon's argument is the `e` typedarray then the variable which captures the error should not be `e`. Maybe it would be better to have better name in the function's argument.